src/pages/user-descriptor/line-graph.jsx

```diff
@@ -1,59 +1,45 @@
 import React from "react";
 import ReactEcharts from "echarts-for-react";
 
-// Expected Object
-function BossObject(Unit, Weight, Latitude, Longitude, Time, Gender) {
-  this.unit = Unit;
-  this.weight = Weight;
-  this.latitude = Latitude;
-  this.longitude = Longitude;
-  this.time = Time;
-  this.gender = Gender;
+// GraphQL
```

Comment:
Don't fix things client side, fix mock server that is returning bad data. Add data that would help your case. So make proper unix timestamps, maybe add more realistic value, etc. The mock server was made quick and dirty, but since you are the main user, you should fix it to meet your needs.